sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -476,7 +477,15 @@ private static DefaultNetworkSpec from(
             builder.portMappings(Collections.emptyMap());
         }
 
-        //TODO(arand) implement network labels
+        if (!Strings.isNullOrEmpty(rawNetwork.getLabelsCsv())) {
+            Map<String, String> labels = new HashMap<>();
+            for (String kv : Arrays.asList(rawNetwork.getLabelsCsv().split(","))) {
+                labels.put(kv.split(":")[0], kv.split(":")[1]);
```

Comment:
Yes! Added validator, confirmed that it will complain at compile time (instead of deployment time).